lib/services/database.go

```diff
@@ -809,6 +809,60 @@ func NewDatabasesFromRDSClusterCustomEndpoints(cluster *rds.DBCluster) (types.Da
 	return databases, trace.NewAggregate(errors...)
 }
 
+// NewDatabasesFromRDSCluster creates all database resources from an RDS Aurora
+// cluster.
+func NewDatabasesFromRDSCluster(cluster *rds.DBCluster) (types.Databases, error) {
```

Comment:
`NewDatabasesFromRDSCluster` and `NewDatabasesFromElastiCacheReplicationGroup` are moved from fetchers  so they can be shared. no logic change.